larray/tests/test_array.py

```diff
@@ -4557,7 +4557,6 @@ def test_diag():
     assert d.i[1] == 1.0
 
 
-@needs_python36
```

Comment:
since python3.6 is now the lowest supported version, you can also drop the needs_python36 decorator everywhere (including its definition)